### PR TITLE
Add GSoD to news

### PR DESCRIPTION
**Pull request by Loquacity**
Adds GSoD. Will need another update once everything is updated and official. @paususe might want to change this.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -231,6 +231,14 @@ <h3>Latest news</h3>
...
         <div class="js-home-carousel home-carousel">
           <!-- News box Start -->
+        <div class="news-box">
+        <div class="news-box-title">
+            <span>15/03/2021</span>
+        </div>
+        <div class="news-box-content">
+            <p>Uyuni is participating in <a href="https://developers.google.com/season-of-docs" target="_blank">Google Season of Docs</a>. If you're a technical writer who would like to contribute to the Uyuni documentation, stay tuned for more information.</p>
+        </div>
+        </div>
```
Comment (juliogonzalez) [inline index.html:241]: Please remove:
```
          <!-- News box Start -->
          <div class="news-box">
            <div class="news-box-title">
              <span>01/03/2021</span>
            </div>
            <div class="news-box-content">
              <p>Uyuni 2021.02 is now available! This mostly a bugfix release including fixes for several <a href="https://saltproject.io/security_announcements/active-saltstack-cve-release-2021-feb-25/" target="_blank">Salt CVES</a> and Ubuntu support for Prometheus exporters' reverse proxy formula. Check the <a href="../pages/stable-version.html#releasenotes">release notes</a> for more details.</p>
            </div>
          </div>
          <!-- News box End -->
```
(251-260). We only keep two news at the same time on this page.

File: pages/news.html [hunk 2]
```diff
@@ -113,7 +114,7 @@ <h1>NEWS</h1>
             <p><strong>02/08/2019</strong> - Uyuni 4.0.2 is now available! This release enables Ubuntu clients, monitoring, content lifecycle management and <a href="../pages/stable-version.html#releasenotes">much more</a>!</p>
             <p><strong>05/04/2019</strong> - openSUSE Summit Nashville - <a href="https://events.opensuse.org/conferences/oSSN19/program/proposals/2276">Managed Monitoring with Prometheus, Grafana and Uyuni</a></p>
             <p><strong>08/03/2019</strong> - SCALE17x Presentation - <a href="https://www.socallinuxexpo.org/scale/17x/presentations/uyuni-adding-flavor-open-source-infrastructure-management">Uyuni: Adding flavor to open source infrastructure managementx</a></p>
-            
+
```
Comment (juliogonzalez) [inline pages/news.html:117]: Why is this needed?

File: index.html [hunk 3]
```diff
@@ -231,6 +231,14 @@ <h3>Latest news</h3>
         </div>
         <div class="js-home-carousel home-carousel">
           <!-- News box Start -->
+        <div class="news-box">
+        <div class="news-box-title">
+            <span>15/03/2021</span>
+        </div>
+        <div class="news-box-content">
+            <p>Uyuni is participating in <a href="https://developers.google.com/season-of-docs" target="_blank">Google Season of Docs</a>. If you're a technical writer who would like to contribute to the Uyuni documentation, stay tuned for more information.</p>
```
Comment (juliogonzalez) [inline index.html:239]: Shouldn't we point to a place where we have ideas for the GSoD? Or tell people how to propose ideas? Just as we did for the GSoC? @paususe?

**Review by juliogonzalez — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (Loquacity) [reply index.html:239]: I'm not 100% certain what the status is, I need Pau to weigh in, because he's organising this. I just wrote the proposals.

File: pages/news.html (hunk 2 above)
Comment (Loquacity) [reply pages/news.html:117]: I was wondering about that myself. My editor doesn't see a change there, so maybe it's the GH diff? :woman_shrugging:

File: index.html (hunk 1 above)
Comment (Loquacity) [reply index.html:241]: Then maybe we shouldn't mention GSoD at all yet? I feel like we don't want to knock the release announce off the front page yet.

Comment (paususe) [reply index.html:241]: We should not add GSoD yet because we have not even applied yet. We do not know whether we will be accepted or not.
